particles/Processing/js/LoadGraphModel.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * @license
+ * Copyright (c) 2019 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+defineParticle(({DomParticle, log, resolver}) => {
+
+  importScripts(resolver(`$here/tf.js`));
+
+  return class extends self.TfMixin(DomParticle) {
+    async update({modelSpec}) {
+      if (modelSpec) {
+        const model = await this.tf.loadGraphModel(resolver(modelSpec.location), modelSpec.options);
```

Comment:
Question: Will the resolver work in test code?

Reply:
What happens if `modelSpec.options` is undefined?

Reply:
`resolver` should work on all platforms, if not it's a bug.